### PR TITLE
first patch

### PR DESCRIPTION
**Pull request by striver01**
Styling done on the UI.

### CODE REVIEW

File: src/pages/Home.js [hunk 1]
```diff
@@ -18,6 +18,16 @@ function Home() {
...
     },
+    {
+      type: "Hollywood Mix",
+      image: `${process.env.PUBLIC_URL}/assets/525661.png`,
+      url: "/hollywood",
+    },
+    {
+      type: "Bollywood Mix",
+      image: `${process.env.PUBLIC_URL}/assets/525661.png`,
+      url: "/bollywood",
```
Comment (Tusharsaxena3112) [inline src/pages/Home.js]: Have you made these routes in App.js?

Comment (striver01) [reply src/pages/Home.js]: Updated it in my next commit but not updated the component part.

Comment (striver01) [reply src/pages/Home.js]: As those pages are not made yet.

**Review by Tusharsaxena3112 — APPROVED** [review]
lgtm

**Pull request merged**